pymongo_auth_aws-1.0.2.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pymongo_auth_aws-1.0.2.dist-info/LICENSE,sha256=xx0jnfkXJvxRnG63LTGOxlggYnIysveWIZ6H3PNdCrQ,11357
pymongo_auth_aws-1.0.2.dist-info/METADATA,sha256=-2qNbXfMs_-z4FTOblcquqIXnNWoNbbIMNpmx4IqGok,4546
pymongo_auth_aws-1.0.2.dist-info/RECORD,,
pymongo_auth_aws-1.0.2.dist-info/WHEEL,sha256=ADKeyaGyKF5DwBNE0sRE5pvW-bSkFMJfBuhzZ3rceP4,110
pymongo_auth_aws-1.0.2.dist-info/top_level.txt,sha256=P21opOPUCOCYmeePHPABoeATMGhdoQqi0BeiIRVr348,17
pymongo_auth_aws/__init__.py,sha256=vVXmWNpo2AelXeZjPr10gU0FKzLB_w3iIS7RP4xki2A,751
pymongo_auth_aws/__pycache__/__init__.cpython-37.pyc,,
pymongo_auth_aws/__pycache__/auth.cpython-37.pyc,,
pymongo_auth_aws/__pycache__/errors.cpython-37.pyc,,
pymongo_auth_aws/__pycache__/version.cpython-37.pyc,,
pymongo_auth_aws/auth.py,sha256=1iuDfs_3CZNC236xDO6pfW8AWX8IjFxG9vGOhAF9CrQ,8587
pymongo_auth_aws/errors.py,sha256=ilEemepO2UD-cW5C7ZjiQWhZCmidDHlSheoWB1_2rro,725
pymongo_auth_aws/version.py,sha256=eCRuoYNVpu6-jHgMDQEgUABUuyuu-mJhkrysGD57e_4,604
